features/ABC999_Alignment-main-points.feature

```diff
@@ -0,0 +1,51 @@
+@implementer-agreement
+@ALB
+Feature: Alignment main points
+
+  Background: Linear elements with main point referents
+
+    Given the template "Alignment Main Points"
+      And any "ReferentObjectType" equals "MainPoint"
+
+  Scenario: Main point relationship names
+    
+    Then all "PositionsName" equals "StartsAt" or "EndsAt"
+
+  Scenario: Segment to main point relationship cardinality
+
+    Given an IfcAlignmentSegment
+     Then number of values for "PositionsName" should be "2"
+      And a value for "PositionsName" should be "StartsAt"
+      And a value for "PositionsName" should be "EndsAt"
+
+  Scenario: Main point to segment relationship cardinality - first
+
+    Given an IfcAlignmentSegment
+      And it is [first] in relationship "Nests"
+      And any "PositionsName" equals "StartsAt"
+      And any "Referent"
+     Then number of values for "PositionsName" should be "1"
```

Comment:
It feels weird to have this many "special tokens" in "natural language"-like specification. I can hardly imagine that [any] domain expert would be able to write in this manner.